src/encoded/types/variant.py

```diff
@@ -348,6 +372,41 @@ def most_severe_location(self, request):
                 break
         return result
 
+    @calculated_property(schema={
+        "title": "Additional Variant Names",
+        "description": "Additional names/aliases this variant is known as",
+        "type": "array",
+        "items": {
+            "type": "string"
+        }
+    })
+    def additional_variant_names(self, request):
+        """This property will allow users to search for specific variants in the filtering tab,
+        using a few different possible variant names.
+         - c. change
+         - p. change (3 letter aa code)
+         - p. change (1 letter aa code)
+        NB: talk to front end about tooltip/click box for example searches
+        """
+        names = []
+        genes = self.properties.get('genes', [])
+        if genes:
+            if genes[0].get('genes_most_severe_hgvsc'):
+                names.append(genes[0]['genes_most_severe_hgvsc'].split(':')[-1])
+            if genes[0].get('genes_most_severe_hgvsp'):
```

Comment:
This comment also wasn't included in the review for some reason. Can we have multiple genes within `genes`? I forget if that's the case or not, but I don't believe it is. If it is, should we include them here?
```suggestion
        for gene in genes:
            if gene.get('genes_most_severe_hgvsc'):
                names.append(gene['genes_most_severe_hgvsc'].split(':')[-1])
            if gene.get('genes_most_severe_hgvsp'):
```

Reply:
I think the way its implemented currently there is only ever one gene, but there is no reason not to add a loop as it would still work if this changes in the future.